reference/user_defined_functions.rst

```diff
@@ -389,6 +415,35 @@ Example
           7 |       7 | new-node |    12345 | default  | f           | t
     (1 row)
 
+master_disable_node
```

Comment:
We didn't discuss this during prioritization, so I thought we wouldn't document this. If we do want to, could you review the function comment here: https://github.com/citusdata/citus/blob/master/src/backend/distributed/utils/node_metadata.c#L171? It has a bit more detail on what it does under the covers. An example is that it also removes reference-table placements from the node.